dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/UserUploadsApi.kt

```diff
@@ -0,0 +1,45 @@
+package org.dataland.datalandbackend.api
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.dataland.datalandbackend.entities.DataMetaInformationForMyDatasets
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+
+/**
+ * Defines the restful dataland-backend API regarding user data exchange
+ */
+@RequestMapping("/users")
```

Comment:
Isn't this endpoint similar to the meta-data endpoint? Why this new path /users?
I for one would expect to get somehow user information like how many users, userIds and so on, under this route.

Reply:
Hmm, I see your point. I am not sure about the perfect way to handle this. However, these endpoints are not returning the usual meta infos but a combination with some company data specifically tailored for the "My Datasets" page. This felt like the natural way of managing this resource because it is identified with a user.

Reply:
ok